### PR TITLE
Fix invalid info about event arguments

### PR DESCRIPTION
**Pull request by Steveb-p**
Events were changed sometime ago and actually have arguments

### CODE REVIEW

File: docs/en/reference/events.rst [hunk 1]
```diff
@@ -203,9 +203,10 @@ listeners:
 
 - 
    Lifecycle Callbacks are methods on the document classes that are
-   called when the event is triggered. They receive absolutely no
-   arguments and are specifically designed to allow changes inside the
-   document classes state.
+   called when the event is triggered. They may receive receive instances
```
Comment (malarzm) [inline docs/en/reference/events.rst]: How about

> They receive instance of ``Doctrine\ODM\MongoDB\Event\LifecycleEventArgs`` as the only argument (see relevant examples below) and are specifically [...]

IMO "may" suggests the argument might not be passed and I think we're always passing it

**Comment by malarzm** [comment]
Ah, would you also mind squashing commits?

File: docs/en/reference/events.rst (hunk 1 above)
Comment (Steveb-p) [inline docs/en/reference/events.rst]: Wasn't sure if all events now receive objects. Will update & squash.

**Comment by Steveb-p** [comment]
Squashed & included your comment. Or the other way around ;)

**Comment by malarzm** [comment]
Thanks a bunch!

**Pull request merged**